src/System.Buffers.Primitives/System/Buffers/IRetainable.cs

```diff
@@ -3,9 +3,10 @@
 
 namespace System.Buffers
 {
-    internal interface IKnown
+    public interface IRetainable
     {
-        void AddReference();
+        void Retain();
```

Comment:
The method is abstract. Some implementations don't actually do reference counting.